.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,31 @@
-- [ ] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-- [ ] The title of this PR follows the `where the change occurs: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
+Complete the following REQUIRED checkboxes:
+<!-- While editing this template, replace the whitespace between the square brackets with an 'x', e.g. [x] -->
+-   [ ] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
+-   [ ] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
 
-<!-- Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete these later if they are not currently applicable. -->
-- [ ] I have previewed all lesson file(s) included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly.
-- [ ] If one exists, I have linked a related open issue to this PR in Step 2 below
-- [ ] If changes were requested, I have made them and re-requested a review from the maintainer (top of the right sidebar)
+Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
+-   [ ] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
+-   [ ] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)
 
-**1. Description of the Changes**
-<!-- A clear and concise description of your changes. If this PR is not related to an open issue also include why you are proposing these changes, such as what benefits the changes have or what problem(s) they solve. --> 
+<hr>
 
+**1. Because:**
+<!--
+If this PR closes an open issue, replace the XXXXX below with the issue number, e.g. Closes #2013. Or if the issue is in another TOP repo replace the #XXXXX with the URL of the issue, e.g. Closes https://github.com/TheOdinProject/curriculum/issues/XXXXX
 
-**2. Related Issue**
-<!-- If the PR is not related to any open issue, skip this step. 
-
-Otherwise, replace the XXXXX with the issue number, e.g. Closes #2013, or if the issue is in another TOP repo replace #XXXXX with the URL of the issue, e.g. Closes https://github.com/TheOdinProject/curriculum/issues/XXXXX -->
+Otherwise, provide a clear and concise reason for your pull request, e.g. what problem it solves or what benefit it provides. If this PR is related to, but does not close, another issue or PR, you can also link it as above without the 'Closes' keyword, e.g. "Related to #2013".
```

Comment:
The use of "otherwise" seems to imply an either or situation here. Is that the intent?

If the PR is related to an open issue and the contributor marks the closing issue number, should they not include a brief description as well?

Reply:
I don't necessarily think a description is needed if the PR closes an open issue. The open issue would technically be the description as to the purpose/benefit of the PR or what problems are solved.

Reply:
Right, I wasn't thinking it was necessary but I was thinking more along the lines of: if I am making a PR that closes an open issue, am I not allowed to write a very brief description here as well? Most of the PRs I've seen are usually doing both the issue link and the brief description. Wondering if this change might throw them off.

One other thought on this section. Since the first line was removed: "A clear and concise description of your changes." And now the first line begins: "If this PR closes an open issue..." Do we think people might skip right over this section if it's not related to an issue? Therefore running the risk of no linked issue and no description?

Reply:
If a description is included in addition to the issue being linked to close, that wouldn't really be a problem. The change may throw people off initially, but this is all in an attempt to make things more consistent across repos; the TOP repo has successfully used a format somewhat similar to this for a while, and after some discussion with Kevin we came to the decision to go with this format.

I think no matter what people will find an excuse to not read, though, and that there's only so much we can do to ensure they read things properly. Between telling users to read the template in its entirety and to read the Contributing Guide thoroughly, if users still don't fill things out correctly then that'll have happened no matter what. That's not to say I don't get where you're coming from, it's just I've seen others try to ensure users read things and they still find a way not to 😆

Reply:
Gotcha! Hopefully all goes smoothly. If we do start getting a bunch of blank responses in that section, another idea would be adding in the word "REQUIRED" the way the checkbox sections do at the top. But people might choose not to see that either!